askama_shared/src/generator.rs

```diff
@@ -876,16 +876,34 @@ impl<'a, S: std::hash::BuildHasher> Generator<'a, S> {
 
         match *var {
             Target::Name(name) => {
-                if !self.locals.contains(&name) {
+                let meta = self.locals.get(&name).cloned();
```

Comment:
Why do we have `.cloned()` here? It doesn't seem necessary...

Reply:
Otherwise the subsequent `self.write_buf_writable(buf)` is not possible, as `self` would be borrowed.

Though I guess I can move `meta.is_none()` and turn it into a variable, next to `let shadowed` if you'd rather have that? :)

Reply:
Let's just keep it. 👍